pod/perlfunc.pod

```diff
@@ -4584,7 +4584,7 @@ Reading from a file:
         or die "Can't open < input.txt: $!";
 
     # Process every line in input.txt
-    while (my $line = <$fh>) {
+    while (my $line = readline($fh)) {
```

Comment:
This should be `while (defined(my $line = readline($fh))) {`  BUt otherwise seems reasonable unless this is documenting `<>` itself.

Reply:
it actually doesn't matter; the implicit defined check is applied to the optree regardless of whether it was parsed from `readline` or `<>`.

Reply:
Oh, i sit corrected:
```
$ perl -MO=Deparse -e'while (my $line = <>) { ... }'
while (defined(my $line = readline ARGV)) {
    do {
        die 'Unimplemented'
    };
}
-e syntax OK
```

thanks.